wp-content/plugins/mycred/addons/sell-content/includes/mycred-sell-functions.php

```diff
@@ -483,7 +483,8 @@ function mycred_sell_content_payment_buttons( $user_id = NULL, $post_id = NULL )
                 //Andy edited
                 //original code:    $button   = '<button type="button" class="mycred-buy-this-content-button ' . $setup['button_classes'] . '" data-pid="' . $post_id . '" data-type="' . $point_type . '">' . $button_label . '</button>';
                 
-				$button       = '<button type="button" class="mycred-buy-this-content-button ' . $setup['button_classes'] . '" data-pid="' . $post_id . '" data-type="' . $point_type . '" onclick="javascript:self.location.reload()";>' . $button_label . '</button>';
+				// $button       = '<button type="button" class="mycred-buy-this-content-button ' . $setup['button_classes'] . '" data-pid="' . $post_id . '" data-type="' . $point_type . '" onclick="javascript:self.location.reload()";>' . $button_label . '</button>';
+                $button       = '<button type="button" class="mycred-buy-this-content-button ' . $setup['button_classes'] . '" data-pid="' . $post_id . '" data-type="' . $point_type . '" onclick="javascript:setTimeout(function(){self.location.reload();}, 500)";>' . $button_label . '</button>';
```

Comment:
Temporarily using setTimeOut() to delay redirection.
Suggest to use async method to handle http request.